ts/features/wallet/component/CardLayoutPreview.tsx

```diff
@@ -60,13 +57,12 @@ const styles = StyleSheet.create({
 });
 
 /**
- * A Generic preview card layout extracted from {@link CardComponent} and simplified.
+ * A preview card layout that generalize {@link CardPreview}.
```

Comment:
```suggestion
 * A preview card layout that generalizes {@link CardPreview}.
```

Reply:
Fixed in [4baa476](https://github.com/pagopa/io-app/pull/2878/commits/4baa4762f718d987ebea2f98a9dbe161067f06ad)